app/src/org/commcare/android/javarosa/PollSensorAction.java

```diff
@@ -49,27 +50,10 @@
     public static final String XPATH_ERROR_ACTION = "poll_sensor_xpath_error_action";
     private TreeReference target;
 
-    private LocationManager mLocationManager;
+    private static LocationManager mLocationManager;
```

Comment:
Has to be static because there will be 2 `PollSensorAction` instances per form when there are `xforms-ready` and `xforms-revalidate` actions